src/components/panel/demoGroups/index.html

```diff
@@ -0,0 +1,44 @@
+<div ng-controller="PanelGroupsCtrl as ctrl" ng-cloak>
+
+  <md-toolbar class="md-accent">
+    <div class="md-toolbar-tools">
+      <md-button
+        class="md-icon-button"
+        aria-label="Settings"
+        ng-click="ctrl.showMenu($event, 'settings', 'toolbar')">
```

Comment:
Demos should be something we expect users to _want_ to do - the common case, not the everything case. All I'm saying in the second paragraph is that the common case is to declare the group in the open method, not pass it in. Most people will have exactly one open method per panel, not one method that's used to open multiple panels.